cin_validator/cin_validator.py

```diff
@@ -284,10 +284,14 @@ def __init__(
         )
         self.full_issue_df.reset_index(drop=True, inplace=True)
 
-        # combine multichild issues
+        # remove header issues from main dataframe and transfer to no-child-id dataframe
         header_issues = self.full_issue_df[
```

Comment:
Is there a reason this isn't `self.header_issues` like the surrounding calculated values? Is there no chance this will ever be used outside of this function?

This is fine if that's the case. I just wanted to flag it just in case it was a mistake.

Reply:
Good shout.
This is the last step in the computation before values are returned from the backend. So there is no chance that another function after this will need `header.issues`.

Reply:
Also, we don't need header issues on their own. We select them out of the main issues df to combine them with la issues which were processed separately. So the combined df, `self.multichild_issues`, is added as an attribute that can be accesses from outside the class.